torch/_inductor/ir.py

```diff
@@ -5471,6 +5468,74 @@ def canonicalize(self):  # type: ignore[no-untyped-def]
         index = sympy_subs(sympy.expand(index), replacement)
         return index, tuple(new_sizes)
 
+    def codegen_unbacked_symbol_defs(self, outputs, wrapper) -> None:  # type: ignore[no-untyped-def]
```

Comment:
1. Move codegen_unbacked_symbol_defs upper level in the inheritance hierarchy from FallbackKernel to ExternKernel to share the logic where we return unbacked symbol output.
2. make outputs explicit.